Add info about setting keyboard layout for new user (#4169)
raspberrypi/documentation: documentation/asciidoc/computers/configuration/users.adoc

If the above command runs successfully, permissions were successfully configured for the user.

> If you've changed your keyboard layout, you may find that the newly-added user is using the "wrong" keyboard layout. To fix this, log in as the new user and run:

Reviewer: ```suggestion
If you've changed your keyboard layout, you might find that the newly-added user is using the "wrong" keyboard layout. To fix this, sign in as the new user and run:
```

> If you've changed your keyboard layout, you might find that the newly-added user is using the "wrong" keyboard layout. To fix this, sign in as the new user and run:

Reviewer 2: ```suggestion
If you've changed your keyboard layout, you might find that the newly added user is using the "wrong" keyboard layout. To fix this, sign in as the new user and run:
```
